### DISCUSSION TITLE
Extracellular Membrane, Intracellular, Interstitial - What Does this mean for the lumen?

### DISCUSSION THREAD

**Discussion by stwiebe** [Open-Systems-Pharmacology/Forum#887] [Questions & Problems] [answered]
Dear OSP community,

I am relatively new to PBPK and am currently trying to implement a simple method to mimic lysosomal trapping in a model I'm working on, using the methods found in this post: #346
[](https://github.com/Open-Systems-Pharmacology/Forum/issues/346) 

When adding an unspecified protein, I have to choose if it is localized interstitially, on the extracellular membrane, or intracellularly:
![image](https://user-images.githubusercontent.com/82565827/114823040-3d720c00-9dc3-11eb-9960-e431c41d1d67.png)

If I'm adding the protein in the lumen, I wonder how the different choices affect the calculations/what the different choices (specifically interstitial and intracellular) would mean, given that the lumen is the space inside? 

Any help anyone can give to help me better understand this (and which option I should then choose) is very much appreciated - thanks in advance!

Sabrina

**Comment by StephanSchaller** [comment, marked as answer]
Dear Sabrina,

this setting only affects the entries fur "tissues". Lumen is not part of the tissue and not affected by this choice.

**Comment by stwiebe** [comment]
Thank you very much for the quick response, Stephan! I’m glad to see it doesn’t make a difference for the lumen, as we were a little confused as to how that would work…
